spec/outputs/elasticsearch_spec.rb

```diff
@@ -73,6 +73,75 @@
     end
   end
 
+  describe "ship lots of events w/ default index_type and fixed routing key", :elasticsearch => true do
+    # Generate a random index name
+    index = 10.times.collect { rand(10).to_s }.join("")
```

Comment:
This should go into a let variable (more info at https://relishapp.com/rspec/rspec-core/v/3-2/docs/helper-methods/let-and-let )